app/build.gradle

```diff
@@ -117,11 +117,11 @@ dependencies {
 	implementation 'androidx.cardview:cardview:1.0.0'
 	implementation 'androidx.recyclerview:recyclerview:1.1.0'
 	implementation 'androidx.legacy:legacy-preference-v14:1.0.0'
-	implementation 'androidx.constraintlayout:constraintlayout:2.0.1'
+	implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
```

Comment:
Updating libraries for a release?

Reply:
This is just contained in the last, follow-up commit. The release should be at commit [`release 2.1.1`](https://github.com/grote/Transportr/pull/736/commits/d76cb30b96bda0e83209076ac645a0e61e080485).

Reply:
Ah, I would prefer follow-up stuff in a dedicated PR.